backend/app/api/endpoints/wework_auth.py

```
@@ -225,6 +400,53 @@ async def approve_wework_auth_session(
     return WeworkAuthSessionActionResponse(status="approved")
 
 
+@router.post("/refresh", response_model=WeworkTokenRefreshResponse)
+async def refresh_wework_access_token(
+    body: WeworkTokenRefreshRequest,
+    request: Request,
+    db: Session = Depends(get_db),
```

Ruff (0.16.2)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)